src/dom/Value.ts

```diff
@@ -106,6 +106,14 @@ export interface Value {
    */
   get(...pathElements: PathElement[]): Value | null;
 
+  /**
+   * For the Struct type, returns an array containing all the values for given field name or field name path
```

Comment:
We should emphasize that this is meant to be used when working with input that contains duplicate fields. Maybe:

```suggestion
   * Like the `get(PathElement[])` method, but returns *all* of the values associated 
   * with a given field name instead of just the last one. For example, for the
   * following struct:
   * 
   *     {
   *         foo: 1,
   *         foo: null,
   *         foo: true,
   *     }
   * 
   * a call to `get("foo")` would return a `dom.Boolean` representing `true`
   * while a call to `getAll("foo")` would return an `Array` of
   * `dom.Value`s: `[1, null, true]`.
```